app/course/services/courseActionCreators.js

```diff
@@ -141,6 +144,24 @@ courseApp.service('courseActionCreators', function (courseStateService, courseSe
 				$rootScope.$emit('toast', { message: "Something went wrong. Please try again.", type: "ERROR" });
 			});
 		},
+		deleteMultipleCourses: function (courseIds, workgroupId, year) {
+			var self = this;
+			courseService.deleteMultipleCourses(courseIds, workgroupId, year).then(function () {
+				$rootScope.$emit('toast', { message: "Deleted courses.", type: "SUCCESS" });
+				var action = {
+					type: DELETE_MULTIPLE_COURSES,
+					payload: {
+						courseIds: courseIds
+					}
+				};
+				courseStateService.reduce(action);
+
+				self.closeCourseDeletionModal();
+				self.deselectAllCourseRows();
+			}, function (err) {
+				$rootScope.$emit('toast', { message: "Something went wrong. Please try again.", type: "ERROR" });
```

Comment:
Let's make these types of messages more specific: "Unable to delete multiple courses. Please try again."